arcgis-ios-sdk-samples/Display information/Graphics overlay (dictionary renderer) 3D/GraphicsOverlayDictionaryRenderer3DViewController.swift

```diff
@@ -38,8 +35,21 @@ class GraphicsOverlayDictionaryRenderer3DViewController: UIViewController {
     func makeGraphicsOverlay() -> AGSGraphicsOverlay {
         let graphicsOverlay = AGSGraphicsOverlay()
         
-        let dictionarySymbolStyle = AGSDictionarySymbolStyle(specificationType: "mil2525d")
-        graphicsOverlay.renderer = AGSDictionaryRenderer(dictionarySymbolStyle: dictionarySymbolStyle)
+        if let styleURL = Bundle.main.url(forResource: "mil2525d", withExtension: "stylx") {
+            let dictionarySymbolStyle = AGSDictionarySymbolStyle(url: styleURL)
+            dictionarySymbolStyle.load { [weak self, weak graphicsOverlay] error in
```

Comment:
My understanding is that this shouldn't be necessary, but it works around an issue where sometimes the graphics overlay's extent is valid, sometimes it isn't.